pkg/webui/console/store/middleware/logics/applications.js

```diff
@@ -22,6 +22,15 @@ import * as link from '@console/store/actions/link'
 
 import createEventsConnectLogics from './events'
 
+const createApplicationLogic = createRequestLogic({
+  type: applications.CREATE_APP,
+  process: async ({ action }) => {
+    const { ownerId, app, isAdmin } = action.payload
+    const newApp = await tts.Applications.create(ownerId, app, isAdmin)
+    return newApp
```

Comment:
```suggestion
    const { ownerId, app, isUserOwner } = action.payload
    
    return await tts.Applications.create(ownerId, app, isUserOwner)
```
Please be more consistent:
1. We usually put one empty line before returning.
2. No need to assign the results to a variable to return them (unless the result gets modified).

There's similar problems in the other logic functions.